repose-aggregator/src/docs/asciidoc/filters/herp.adoc

```diff
@@ -1,3 +1,213 @@
+                    typeURI="http://schemas.dmtf.org/cloud/audit/1.0/event"
+                    eventTime="{{cadfTimestamp timestamp}}"
+                    action="{{cadfMethod requestMethod}}"
+                    outcome="{{cadfOutcome responseCode}}">
+            <cadf:initiator id="{{#if impersonatorName}}impersonatorName{{else}}{{userName}}{{/if}}"
+                            typeURI="network/node" name="{{#if impersonatorName}}impersonatorName{{else}}{{userName}}{{/if}}">
+                <cadf:host address="{{requestorIp}}" agent="{{userAgent}}" />
+            </cadf:initiator>
+            <cadf:target id="{{targetHost}}" typeURI="service" name="{{serviceCode}}" >
+                <cadf:host address="{{targetHost}}" />
+            </cadf:target>
+            <cadf:attachments>
+                <cadf:attachment name="auditData" contentType="ua:auditData">
+                    <cadf:content>
+                        <ua:auditData version="1">
+                            <ua:region>{{region}}</ua:region>
+                            <ua:dataCenter>{{dataCenter}}</ua:dataCenter>
+                            <ua:methodLabel>{{methodLabel}}</ua:methodLabel>
+                            <ua:requestURL>{{requestURL}}</ua:requestURL>
+                            <ua:queryString>{{requestQueryString}}</ua:queryString>
+                            <ua:tenantId>{{defaultProjectId}}</ua:tenantId>
+                            <ua:responseMessage>{{responseMessage}}</ua:responseMessage>
+                            <ua:userName>{{userName}}</ua:userName>
+                            <ua:roles>{{#each roles}}{{#if @index}} {{/if}}{{.}}{{/each}}</ua:roles>
+                        </ua:auditData>
+                    </cadf:content>
+                </cadf:attachment>
+            </cadf:attachments>
+            <cadf:observer id="{{serviceCode}}-{{clusterId}}-{{nodeId}}" name="repose" typeURI="service/security" />
+            <cadf:reason reasonCode="{{responseCode}}"
+                         reasonType="http://www.iana.org/assignments/http-status-codes/http-status-codes.xml"/>
+        </cadf:event>
+        ]]>
+    </template>
+    <filterOut>
+        <match field="userName" regex=".*[fF]oo.*"/>
+        <!-- Implicit AND -->
+        <match field="dataCenter" regex="DFW"/>
+    </filterOut>
+    <!-- Implicit OR -->
+    <filterOut>
+        <match field="userName" regex=".*[bB]ar.*"/>
+        <!-- Implicit AND -->
+        <match field="parameters.abc" regex="123"/>
+    </filterOut>
+</highly-efficient-record-processor>
+----
+
+[NOTE]
+The `{{methodLabel}}` variable value is provided by the `api-validator` filter.
+If you aren't using that filter, `{{methodLabel}}` will return empty string.
+
+=== User Access Event Recording With Flume
+For information about using the Flume with the HERP filter for user access event recording, see link:../recipes/api-event-flume.html[CF Flume Sink].
+
+=== Configurable parameters
```

Comment:
I don't think we want this section at all? Example and the schema link should suffice.